packages/contracts-bedrock/justfile

```diff
@@ -61,26 +61,30 @@ export pinnedBlockNumber := env_var_or_default('FORK_BLOCK_NUMBER', '21387830')
 print-pinned-block-number:
   echo $pinnedBlockNumber
 
-# Runs upgrade path variant of contract tests.
+# Prepares the environment for upgrade path variant of contract tests and coverage.
 # Env Vars:
 # - ETH_RPC_URL must be set to a production (Sepolia or Mainnet) RPC URL.
 # - FORK_BLOCK_NUMBER can be set in the env, or else will fallback to the default block number.
 #   Reusing the default block number greatly speeds up the test execution time by caching the
 #   rpc call responses in ~/.foundry/cache/rpc. The default block will need to be updated
 #   when the L1 chain is upgraded.
 # TODO(opcm upgrades): unskip the "NMC" tests which fail on the forked upgrade path with "UnexpectedRootClaim" errors.
-test-upgrade *ARGS: build-go-ffi
+prepare-upgrade-env *ARGS : build-go-ffi
   #!/bin/bash
   echo "Running upgrade tests at block $pinnedBlockNumber"
   export FORK_BLOCK_NUMBER=$pinnedBlockNumber
   export NO_MATCH_CONTRACTS="OptimismPortal2WithMockERC20_Test|OptimismPortal2_FinalizeWithdrawal_Test|'AnchorStateRegistry_*'|FaultDisputeGame_Test|PermissionedDisputeGame_Test|FaultDispute_1v1_Actors_Test|DelayedWETH_Hold_Test"
   export NO_MATCH_PATHS="test/dispute/AnchorStateRegistry.t.sol"
-  FORK_RPC_URL=$ETH_RPC_URL \
-    FORK_TEST=true \
-    forge test --match-path "test/{L1,dispute}/**" \
-    --no-match-contract "$NO_MATCH_CONTRACTS" \
-    --no-match-path "$NO_MATCH_PATHS" \
-    {{ARGS}}
+  FORK_RPC_URL=$ETH_RPC_URL
+  FORK_TEST=true
+  {{ARGS}} \
```

Comment:
instead of this `{{ARGS}}` here, which I think gets a bit confusing to follow, could we export all the env vars, and then write `test-upgrade` like this?:
```
test-upgrade *ARGS: just prepare-upgrade-env
 forge test {{ARGS}}
```